source/conf.py

```diff
@@ -409,7 +409,7 @@ def setup(app):
 .. raw:: html
 
   <link rel="stylesheet" href="https://media.readthedocs.org/css/sphinx_rtd_theme.css" type="text/css" />
-  <link rel="stylesheet" href="http://developers.invoca.net/en/""" + version+  """/_static/css/custom.css" type="text/css" />
+  <link rel="stylesheet" href="http://developers.invoca.net/en/""" + version +  """/_static/css/custom.css" type="text/css" />
```

Comment:
the way to do this (for all three of these resources here) is to remove the protocol entirely and the browser will load based on what the page is loaded as:

```
<link rel="stylesheet" href="//media.readthedocs.org/..."
<link rel="stylesheet" href="//developers.invoca.net/en..."
<link rel="stylesheet" href="//media.readthedocs.org/..."
```